cmd/geoipupdate/defaults_win.go

```diff
@@ -0,0 +1,14 @@
+// +build windows
+
+package main
+
+// These can be changed using compile time flags. e.g.,
+//
+// go build -ldflags "-X main.defaultConfigFile=/usr/local/etc/GeoIP.conf
+// -X main.defaultDatabaseDirectory=/usr/local/share/GeoIP"
+var (
+	// I'm not sure these make sense. However they can be overridden at runtime
+	// and in the configuration, so we have some flexibility.
+	defaultConfigFile        = `C:\ProgramData\GeoIP.conf`
+	defaultDatabaseDirectory = `C:\ProgramData\GeoIP`
```

Comment:
I am guessing `os.GetEnv("SYSTEMDRIVE") + '\ProgramData\MaxMind\GeoIP'` would be a more appropriate directory.

Reply:
Sounds good, thanks!